static/app/views/settings/account/mergeAccounts.tsx

```diff
@@ -0,0 +1,343 @@
+          {
+            strong: <strong />,
+          }
+        )}
+      </TextBlock>
+      <TextBlock>{t(`Your currently active account:`)}</TextBlock>
+      <UserContext value>
+        <Users users={currentAccount} onSelect={onSelect} />
+      </UserContext>
+      <TextBlock>{t(`Your other accounts:`)}</TextBlock>
+      <UserContext value={false}>
+        <Users users={otherAccounts} onSelect={onSelect} />
+      </UserContext>
+    </Fragment>
+  );
+}
+
+type RenderSelectAccountsProps = {
+  onSelect: (newUserId: string) => void;
+  users: UserWithOrganizations[];
+};
+
+type UserRowProps = {
+  id: string;
+  lastSeen: string;
+  name: string;
+  onSelect: (newUserId: string) => void;
+  organizations: string;
+};
+
+type UserProps = {
+  onSelect: (newUserId: string) => void;
+  users: UserWithOrganizations[];
+};
+
+function Users({users, onSelect}: UserProps) {
+  const isPrimaryUser = useContext(UserContext);
+  if (isPrimaryUser) {
+    return (
+      <Panel>
+        <UserPanelHeader>
+          <div>{t('Name')}</div>
+          <div>{t('Last Active')}</div>
+          <div>{t('Organizations')}</div>
+        </UserPanelHeader>
+        <PanelBody>
+          {users.map(userObj => (
+            <UserRow
+              onSelect={onSelect}
+              key={userObj.id}
+              id={userObj.id}
+              name={userObj.name}
+              lastSeen={userObj.lastActive}
+              organizations={userObj.organizations.join(', ')}
+            />
+          ))}
+        </PanelBody>
+      </Panel>
+    );
+  }
+  return (
+    <Panel>
+      <UserPanelHeader>
+        <div>{t('Name')}</div>
+        <div>{t('Last Active')}</div>
+        <div>{t('Organizations')}</div>
+        <div>{t('Merge')}</div>
+      </UserPanelHeader>
+      <PanelBody>
+        {users.map(userObj => (
+          <UserRow
+            onSelect={onSelect}
+            key={userObj.id}
+            id={userObj.id}
+            name={userObj.name}
+            lastSeen={userObj.lastActive}
+            organizations={userObj.organizations.join(', ')}
+          />
+        ))}
+      </PanelBody>
+    </Panel>
+  );
+}
+
+function UserRow({id, name, lastSeen, organizations, onSelect}: UserRowProps) {
+  const isPrimaryUser = useContext(UserContext);
+
+  return (
+    <UserPanelItem>
+      <Name>{name}</Name>
+      {isPrimaryUser ? (
+        'Currently active'
+      ) : lastSeen === '' ? (
+        'Never'
+      ) : (
+        <div>
+          <StyledTimeSince date={lastSeen} />
+        </div>
+      )}
+      <div> {organizations} </div>
+      {isPrimaryUser ? null : (
+        <div>
+          <input
+            type="checkbox"
+            name="user"
+            value={name}
+            onChange={() => onSelect(id)}
+            style={{margin: 5}}
+          />
+        </div>
+      )}
+    </UserPanelItem>
+  );
+}
+
+const tableLayout = `
```

Comment:
needs the css template tag
```suggestion
const tableLayout = css`
```